src/lib.rs

```diff
@@ -274,12 +274,15 @@ where
                 config.inbound_router_capacity,
                 config.inbound_router_max_idle_age,
             );
-            let tls_settings = config.tls_settings.as_ref().map(|settings| {
-                tls::ConnectionConfig {
-                    identity: settings.pod_identity.clone(),
-                    config: tls_server_config
-                }
-            });
+            let tls_settings: tls::ConditionalConnectionConfig<tls::ServerConfigWatch> =
```

Comment:
Is the type annotation necessary here?

Reply:
It was at one point but isn't any longer. Removed in a7e6c10765b946fbe2cd8e4e5b6b686d8d12db2f.